src/stores/user.js

```diff
@@ -36,6 +39,25 @@ export const useUserStore = defineStore('user', {
         this.router.go(0)
       }
     },
+    /**
+     * Create and store a unique Browser id in the storage and in the store
+     * @returns {Promise<void>}
+     */
+    async loadBrowserId() {
+      if (this.getUser && this.getUser.id && this._browserId !== this.getUser.id) {
```

Comment:
I think here you could do just a `if (this.isAuthenticated) { ...`
Can you please check, I think this can improve readability.

Reply:
I wanted to select only line 47

Reply:
Yes for the first part but i need to check also if the current user is not set in the browser id. Th fonction is called many times and we don't need to update the browser id and the storage every time.

 `if (this.isAuthenticated && this._browserId !== this.getUser.id) `

Reply:
Oh true, I missed the second part.